test/rx_ext/serial_disposable_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+abstract class _Disposer {
+  Future<void> dispose(int value);
+}
+
+class _DisposerMock extends Mock implements _Disposer {}
+
+void main() {
+  group('Serialdisposable', () {
```

Comment:
Use "$SerialDisposable" instead. More proof to potential refactorings (renames).

Reply:
I agree that $ makes more sense but it gives problems running tests in VSCode  ( it fails at first after rerun it succeeds )

Reply:
Are you sure you want to use _that_ ~~IDE~~ editor?.. :D